test/unit/pdf_find_controller_spec.js

```diff
@@ -271,5 +271,90 @@ describe("pdf_find_controller", function () {
       pageMatches: [[19, 48, 66]],
       pageMatchesLength: [[8, 8, 8]],
     });
+
+    await testSearch({
+      eventBus,
+      pdfFindController,
+      parameters: {
+        query: "1/2",
```

Comment:
Would it perhaps not be more "interesting" to search for `½` instead here?

Reply:
I'll add one for `½` but I think the `1/2` is good too because the query is 3 chars when the highlight itself contains one char.